nssp_demo/batch/setup_prod_job.py

```diff
@@ -0,0 +1,185 @@
+        Name of the container to use for the job.
+        This container should exist within the Azure
+        Container Registry account associated to
+        the job. Default 'pyrenew-hew'.
+        The container registry account name and enpoint
+        will be obtained from local environm variables
+        via a :class``azuretools.auth.EnvCredentialHandler`.
+
+    container_image_version
+        Version of the container to use. Default 'latest'.
+
+    Returns
+    -------
+    None
+    """
+    supported_diseases = ["COVID-19", "Influenza"]
+
+    disease_list = diseases.split()
+    for d in disease_list:
+        if d not in supported_diseases:
+            raise ValueError(
+                f"Unsupported disease '{d}'; supported diseases "
+                f"are {supported_diseases}"
+            )
+
+    creds = EnvCredentialHandler()
+    client = get_batch_service_client(creds)
+    job = models.JobAddParameter(
+        id=job_id,
+        pool_info=models.PoolInformation(pool_id=pool_id),
+    )
+    create_job_if_not_exists(client, job, verbose=True)
+
+    container_image = (
+        f"{creds.azure_container_registry_account}."
+        f"{creds.azure_container_registry_domain}/"
+        f"{container_image_name}:{container_image_version}"
+    )
+    container_settings = get_container_settings(
+        container_image,
+        working_directory="containerImageDefault",
+        mount_pairs=[
+            {
+                "source": "nssp-etl",
+                "target": "/pyrenew-hew/nssp_demo/nssp-etl",
+            },
+            {
+                "source": "nssp-archival-vintages",
+                "target": "/pyrenew-hew/nssp_demo/nssp-archival-vintages",
+            },
+            {
+                "source": "prod-param-estimates",
+                "target": "/pyrenew-hew/nssp_demo/params",
+            },
+            {
+                "source": "pyrenew-test-output",
+                "target": "/pyrenew-hew/nssp_demo/private_data",
+            },
+        ],
+    )
+
+    base_call = (
+        "/bin/bash -c '"
+        "python nssp_demo/forecast_state.py "
+        "--disease {disease} "
+        "--state {state} "
+        "--n-training-days 75 "
+        "--n-warmup 1000 "
+        "--n-samples 500 "
+        "--facility-level-nssp-data-dir nssp_demo/nssp-etl/gold "
+        "--state-level-nssp-data-dir "
+        "nssp_demo/nssp-archival-vintages/gold "
+        "--param-data-dir nssp_demo/params "
+        "--output-data-dir nssp_demo/private_data "
+        "--report-date {report_date} "
+        "--exclude-last-n-days 5 "
+        "--score "
+        "--eval-data-path "
+        "nssp_demo/nssp-archival-vintages/latest_comprehensive.parquet"
+        "'"
+    )
+
+    locations = pl.read_csv(
+        "https://www2.census.gov/geo/docs/reference/state.txt", separator="|"
+    )
+
+    excluded_locations = ["AS", "GU", "MO", "MP", "PR", "UM", "VI", "WY"]
+
+    all_locations = (
```

Comment:
Given repeated code across scripts maybe we should consider adding some small functions

Reply:
agree, I think `pyrenew_hew.util` makes sense. Out of scope for now though I think.